echo-pipelinetriggers/src/test/groovy/com/netflix/spinnaker/echo/test/RetrofitStubs.groovy

```diff
@@ -75,7 +75,7 @@ trait RetrofitStubs {
       tag = inTag
     }
     def res = new DockerEvent()
-    res.content = new DockerEvent.Content("registry", "repository", tag, "sha")
+    res.content = new DockerEvent.Content("registry", "registry", "repository", tag, "sha")
```

Comment:
What was `registry` bound to before? If it was already present, do we need the `DockerEvent.Content` model change?

Reply:
It was, and still is, bound to `account` (so using the string "registry" as its value is arguably confusing).